### PR TITLE
desktop-lite: fixed qemu container

### PR DESCRIPTION
**Pull request by mrkbac**
`pidof` does not work as intended when using, a qemu or rosetta container, `pgrep -x` has the desired effect.

### CODE REVIEW

File: src/desktop-lite/install.sh [hunk 1]
```diff
@@ -299,9 +299,9 @@ startInBackgroundIfNotRunning()
 {
     log "Starting \$1."
```
Comment (samruddhikhandale) [inline src/desktop-lite/install.sh:300]: Can we bump the patch version in here? thanks! 

https://github.com/devcontainers/features/blob/883cf50eb84f2ad3dada748401a5ad6c06bf43b6/src/desktop-lite/devcontainer-feature.json#L3

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Left a comment. Besides that LGTM, thanks!

**Comment by mrkbac** [comment]
@microsoft-github-policy-service agree

**Review by samruddhikhandale — APPROVED** [review]
Nice!

**Pull request merged**